### DISCUSSION TITLE
"support.apple.com/mac/startup" error on installer boot; latest pulled Monterey and Ventura installers. Latest UEFI DBX.

### DISCUSSION THREAD

**Discussion by thetechhimself** [osy86/HaC-Mini#1098] [Q&A]
Good afternoon folks,

I've got a nice presently, Mojave, and formerly had Catalina, Hades Canyon conversion via the HaC-Mini (thanks again btw). Trying to install Monterey or Ventura on a secondary SSD. Removed my SK P41 w/Ubuntu 22.04, and put in a 2TB P44 Pro, fresh in it's place (I've got Mojave on one SSD, Ubuntu on the other).

Did the usual wipe the SD card (using an SD card instead as it circumvents some of the boot-problems using USB drives for me), ran 3.3.1 on it with defaults. Disabled my Mojave SSD, enabled the new SSD (toggling between SSDs in BIOS) and the system see's both Ventura or Monterey installers I've made with 3.3.1. Tries to boot, I get the Apple Logo, some loading bar, but gets stuck with the prohibitory sign...

![macos-big-sur-startup-screen-prohibitory-sign](https://github.com/osy/HaC-Mini/assets/111404831/72590ca1-f781-45a2-89d0-22d002be6e94)

Tried both Monterey and Ventura. When I try enabling secure boot on the installer itself, it'll reboot on me after a short period, obviously go back to the boot picker, rinse and repeat.

So problem of the day, I can't install either Monterey or Ventura on this thing.

Is this due to needing to enable Secure boot on the installer itself, and then doing something else after (that I'm not)? I did see some unanswered Q&As that sound similar about the prohibitory sign. Or is this a new security patch in the installers that breaks 3.3.1 as I'm using installers I just pulled down this week for both Monterey and Ventura?  Or, is it possible I have a "new" problem? I noted the latest Ubuntu updates, updated my NUC's UEFI revocation trust (DBX), but presumably if that was the issue, I should never be able to get into the installer to begin with, I think.

Any thoughts or advice? I'd really like to get Monterey or Ventura online but I can't make this thing work. NVRAM resets don't help, WiFi disabled, single SSD enabled via BIOS. You got me... Maybe try a USB drive installer again? Those seem to make UEFI on my system grumpy, and I'm concerned about resetting my UEFI trusts on this thing possibly breaking my precious Mojave drive...

**Comment by matvlato** [comment]
Hi there 
I have a similar problem. I patched a 2009 macpro to 5,1 and installed mojave on it, everything worked fine, now I tried to put sonoma but it gives me this message at startup: support.apple.com/mac/startup
